src/pages/EditSingleTripPage/interfaces.d.ts

```diff
@@ -0,0 +1,23 @@
+export interface EditSingleTripPageProps {
+  editTrip: (trip) => Promise<any>;
+  displaySnackMessage?: (message) => Promise<any>;
+  error?: object;
+}
+
+export interface EditSingleTripPageState {
+  isLoading: boolean;
+  isValid: boolean;
+  focused: boolean;
+  fields: {
+    [key: string]: string | number
+  };
+  errors: {
+    [key: string]: string
+  };
+  tripToEdit: {
+    origin: string;
+    destination: string;
+    departure_date: string;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'departure_date' is not in camel case. (camelcase)](https://app.codacy.com/app/mashafrancis/guacamole-fe/pullRequest?prid=3823958)